utils/getInvoices/tests/getInvoices.water-part.test.ts

```diff
@@ -240,9 +240,43 @@ describe('getInvoices - WATER-PART', () => {
         company,
       })
 
-      expect(invoices).not.toContainEqual({
-        type: ServiceType.WaterPart,
+      expect(invoices).not.toContainEqual(
+        expect.objectContaining({ type: ServiceType.WaterPart })
+      )
+    })
+    it('should load when service = { waterPriceTotal: 10 }, company = { waterPart: 10 }', () => {
+      const service: Partial<IService> = {
+        waterPriceTotal: 10,
+      }
+      const company: Partial<IRealestate> = {
+        waterPart: 10,
+      }
+
+      const invoices = getInvoices({
+        service,
+        company,
       })
+
+      expect(invoices).toContainEqual(
+        expect.objectContaining({ type: ServiceType.WaterPart })
+      )
```

Comment:
We need to test the exact result, not only is `WaterPart` presented but also abd it's values